src/lib/backend-api.ts

```diff
@@ -0,0 +1,14 @@
+import { tryFetchingPublic } from './fetching-util';
+import { getBackendUrl, endpoints } from './backend-util';
+/**
+ * Requests to change the power state of the plug
+ * @param {boolean} power the power state the plug should be switched to
+ * @returns {Promise<Response>} the backend response as a {@link Promise<Response>}. Consult backend API documentation for response code semantics (https://github.com/solid-stack-solutions/terralux-backend)
+ */
+async function changePlugState(power: boolean) {
```

Comment:
Ich bekomme "JSDoc types may be moved to TypeScript types."
```suggestion
/**
 * Requests to change the power state of the plug
 * @param power the power state the plug should be switched to
 * @returns the backend response as a {@link Promise<Response>}. Consult backend API documentation for response code semantics (https://github.com/solid-stack-solutions/terralux-backend)
 */
async function changePlugState(power: boolean): Promise<Response> {
```

Reply:
Ah I see. Würde aber eher nicht JSDoc verwenden wollen, einfach nur TS.